eslint.config.mjs

```diff
@@ -0,0 +1,147 @@
+import globals from "globals";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default [
+  {
+    ignores: ["node_modules/*", "_locales/*", ".github/workflows/*", "addons-l10n/*", "libraries/thirdparty/*"],
+  },
+  ...compat.extends("eslint:recommended"),
+  {
+    files: ["**/*.js"],
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        scratchAddons: "writable",
+        __scratchAddonsTraps: "writable",
+        __scratchAddonsRedux: "writable",
+        _realConsole: "writable",
+        __scratchAddonsTextColor: "writable",
+        Vue: "readonly",
+        SparkMD5: "readonly",
+        scratchblocks: "readonly",
+        tinycolor: "readonly",
+        Chart: "readonly",
+        Comlink: "readonly",
+        idb: "readonly",
+        Scratch: "readonly",
+        Blockly: "readonly",
+        Clipboard: "readonly",
+        ClipboardItem: "readonly",
+      },
+
+      ecmaVersion: 2020,
```

Comment:
The oldest browser we now support is Chrome 96 from November 2021.
```suggestion
      ecmaVersion: 2021,
```
The comment near the end of the file should also be updated.

Reply:
Is there the information for the minimum version of Firefox, just so I can do it in one go?

Reply:
It's much more recent, 109 from January 2023.